test/metric_value_benchmark/base_test.go

```diff
@@ -34,6 +35,10 @@ type TestRunner struct {
 	testRunner ITestRunner
 }
 
+type BaseTestRunner struct {
+	*metric.MetricFetcherFactory
```

Comment:
Should it be `IECSTestRunner` ?

Reply:
which one? why? This is currently used for ec2 tests. Note: my other comment about later plan to merge & why I haven't merged the two